crates/nu-command/tests/commands/url/join.rs

```diff
@@ -366,3 +366,22 @@ fn url_join_with_fragment_and_params() {
         "http://usr:pwd@localhost:1234?par_1=aaa&par_2=bbb#frag"
     );
 }
+
+#[test]
+fn url_join_with_empty_params() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+            r#"
+            {
+                "scheme": "https",
+                "host": "localhost",
+                "path": "/foo",
+                "params": {}
+            } | url join
+            "#
+        )
+    );
+
+    assert_eq!(actual.out, "https://localhost/foo");
+    assert_ne!(actual.out, "https://localhost/foo?")
```

Comment:
here, if we get to the `assert_ne` line, this means that `actual.out` **is** equal to `"https://localhost/foo"` and thus it's **impossible** for this variable to equal `"https://localhost/foo?"` :smirk: 

i think this second `assert` is redundant :relieved:

Reply:
Removed.